File: NewExec/CFOUNDNODE.vb
Imports DBiGraphicObjs.DBiGraphicObjects
Imports System

Namespace SaBan
	Public Class CFOUNDNODE
		Private m_FoundObject As GraphicObject

		Private m_NodeIndex As Integer

		Public Property FoundObject() As GraphicObject
			Get
				Return Me.m_FoundObject
			End Get
			Set(value As GraphicObject)
				Me.m_FoundObject = value
			End Set
		End Property

		Public Property NodeIndex() As Integer
			Get
				Return Me.m_NodeIndex
			End Get
			Set(value As Integer)
				Me.m_NodeIndex = value
			End Set
		End Property
	End Class
End Namespace

File: SaBan.Settings/CFOUNDNODE.cs
using DBiGraphicObjs.DBiGraphicObjects;
using System;

namespace SaBan
{
    public class CFOUNDNODE
    {
        private GraphicObject m_FoundObject;

        private int m_NodeIndex;

        public GraphicObject FoundObject
        {
            get
            {
                return this.m_FoundObject;
            }
            set
            {
                this.m_FoundObject = value;
            }
        }

        public int NodeIndex
        {
            get
            {
                return this.m_NodeIndex;
            }
            set
            {
                this.m_NodeIndex = value;
            }
        }
    }
}
